Add warnings for meta descriptions (#1346)
viamrobotics/docs: docs/components/arm/xarmlite.md

weight: 34
type: "docs"
description: "Configure an xArmLite arm."

> metaDescription: "Configure an xArmLite arm to integrate a UFACTORY Lite 6: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."

Author: ```suggestion
metaDescription: "Configure a UFACTORY Lite 6 arm: Navigate to the Create component menu. Enter a name for your arm, select the type arm, and select the xArmLite model. Click Create component. Edit and fill in the attributes as applicable."
```

Reviewer: I agree with this suggestion! Let's merge it in!

tags: ["arm", "components"]
# SMEs: William Spies
---

> Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) for your robot:

Author: ```suggestion
Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) into your robot:
```

> Configure an `xArmLite` arm to integrate a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) for your robot:

Reviewer: ```suggestion
Configure an `xArmLite` arm to add a [UFACTORY Lite 6](https://www.ufactory.cc/product-page/ufactory-lite-6/) to your robot:
```
